src/content/developers/docs/networking-layer/index.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Networking layer
+description: An introduction to Ethereum's networking layer.
+lang: en
+sidebar: true
+sidebarDepth: 2
+---
+
+Ethereum's networking layer is the stack of protocols that allows nodes to find each other and exchange information. After [The Merge](/upgrades/merge/), there will be two parts of client software (execution clients and consensus clients), each with its own distinct networking stack. As well as communicating with other Ethereum nodes, the execution and consensus clients have to communicate with each other. This page gives an introductory explanation of the protocols that enable this communication.
+
+## Prerequisites {#prerequisites}
+
+Some knowledge of Ethereum [nodes and clients](/src/content/developers/docs/nodes-and-clients/) will be helpful for understanding this page.
+
+## The Execution Layer {#execution-layer}
+
+The execution layer's networking protocols is divided into two stacks:
+
+- the discovery stack: built on top of UDP and allows a new node to find peers to connect to
+
+- the DevP2P stack: sits on top of TCP and enables nodes to exchange information
+
+Both stacks work in parallel. The discovery stack feeds new network participants into the network, and the DevP2P stack enables their interactions.
+
+### Discovery {#discovery}
+
+Discovery is the process of finding other nodes in network. This is bootstrapped using a small set of bootnodes (nodes whose addresses are [hardcoded](https://github.com/ethereum/go-ethereum/blob/master/params/bootnodes.go) into the client so they can be found immediately and connect the client to peers). These bootnodes only exist to introduce a new node to a set of peers - this is their sole purpose, they do not participate in normal client tasks like syncing the chain, and they are only used the very first time a client is spun up.
+
+The protocol used for the node-bootnode interactions is a modified form of [Kademlia](https://medium.com/coinmonks/a-brief-overview-of-kademlia-and-its-use-in-various-decentralized-platforms-da08a7f72b8f) which uses a distributed hash table to share lists of nodes. Each node has a version of this table containing the information required to connect to its closest peers. This 'closeness' is not geographical - distance is defined by the similarity of the node's ID. Each node's table is regularly refreshed as a security feature. For example, in the [Discv5](https://github.com/ethereum/devp2p/tree/master/discv5), discovery protocol nodes are also able to send 'ads' that display the subprotocols that the client supports, allowing peers to negotiate about the protocols they can both use to communicate over.
```

Comment:
yea I've never been able to get an unambiguous answer about the scope of DevP2P. In my opinion although discv5 protocol is part of devp2p stack the networking layer is still best considered to be 3 distinct layers: discovery, devp2p and subprotocols because this divides the whole layer up cleanly by function (find peers, establish and maintain secure connections, implement ethereum protocols).